ert3/config/_ensemble_config.py

```diff
@@ -41,6 +41,7 @@ class Input(_EnsembleConfig):
     record: str
     mime: str = ""
     is_directory: Optional[bool]
+    smry_keys: Optional[List[str]]
```

Comment:
Should we have as discussion on whether or not this is the time to introduce transformation configuration? We've discussed this in the past to avoid an explosion of keys and combinations.

Reply:
I have looked a bit into it, and I think it would be really nice to have this merge before we make an attempt at implementing this: https://github.com/samuelcolvin/pydantic/pull/2336